config/factions/dungeon.json

```diff
@@ -148,8 +148,8 @@
 			"mageGuild" : 5,
 			"primaryResource" : "sulfur",
 			"warMachine" : "ballista",
-			"moatDamage" : 90,
-			"moatHexes" : [ 11, 28, 44, 61, 77, 111, 129, 146, 164, 181 ],
+			"moatAbility" : "core:spell.dungeonMoat",
```

Comment:
Missed this during review, but preferred form here is
`"moatAbility" : "dungeonMoat",`
1) No need to specify mod ID for requests within same mod
2) No need to specify identifier type (spell) for requests that only accept specific type

Reply:
Okay, I just thought than more verbose is better for modmakers.
Will do short form later.

Reply:
Will do this myself soon - partially to get rid of weird and illogical error messages when provided ID has unexpected form

Reply:
@IvanSavenko why we cannot accept all form of valid ID?

Reply:
Will fix that part soon, so all valid ID's should always be accepted.

As to why we have it now this way - probably because nobody bothered to fix it?
My initial design was to have minimalistic ID's - e.g. don't specify ID type unless required & don't specify target mod unless required.

Specifying target mod scope was added because some mods had objects with same ID. And this is only scenario when you need to specify target mod to resove ID.
Specifying ID type is only necessary in some generic cases like bonuses. All other usages of ID work without type.